solver/src/main.rs

```diff
@@ -169,14 +169,15 @@ async fn main() {
     .expect("failed to create gas price estimator");
 
     // TODO - Currently we use only Uniswap for price estimation.
-    let price_estimator = Arc::new(UniswapPriceEstimator::new(
+    let uniswap_pair_provider = Arc::new(UniswapPairProvider {
```

Comment:
Let's also use Sushiswap here.

Reply:
I think I have already made this change... Is it possible this comment is already outdated?

Reply:
Yes. Sushiswap is directly below